site/src/models/nfts.js

```diff
@@ -28,7 +49,13 @@ export const has = async (key) => {
 export const set = async (key, value, options) => {
   const savedValue = await get(key)
   if (savedValue === null) {
-    await stores.nfts.put(encodeKey(key), JSON.stringify(value))
+    const kvKey = encodeKey(key)
+    await stores.nfts.put(kvKey, JSON.stringify(value))
+    await stores.nftsIndex.put(
+      encodeIndexKey({ ...key, created: new Date(value.created) }),
```

Comment:
we should probably make an issue to change all these Dates to numbers and stop this back and forth between Date and number (ms)

Reply:
https://github.com/ipfs-shipyard/nft.storage/issues/67